### PR TITLE
Expose Api also via OCS

### PR DESCRIPTION
**Pull request by to-kn**
refs #808

### CODE REVIEW

**Comment by to-kn** [comment]
With these changes, the groupfolders API can be accessed via:

curl -X POST "https://username:password@domain-name/v2.php/apps/groupfolders/folders" -d "mounpoint:testfolder" -H "OCS-APIRequest:true"

File: appinfo/routes.php [hunk 1]
```diff
@@ -52,6 +52,73 @@
 		'verb' => 'POST'
 	],
 
+	[
```
Comment (juliushaertl) [inline appinfo/routes.php:55]: There is no need to have those as regular routes, since Nextcloud can just reuse the OCS endpoints.

**Comment by to-kn** [comment]
@juliushaertl you're welcome! Thank you for your comment - did i get it right, that you are suggesting to remove the 'routes' entry in favor of the 'ocs' entry?

**Comment by juliushaertl** [comment]
Just the additional entries you added to the routes sections for now. We can also migrate the existing routes to ocs endpoints, but that requires adjustments in the frontend code that queries them, so I'd say this is something for a separate pull request.

**Comment by juliushaertl** [comment]
I cherry-picked your commit to https://github.com/nextcloud/groupfolders/pull/997 and addressed my comments regarding an all-ocs-api. Thanks for getting this started.

**Pull request closed without merging**